### PR TITLE
settings: add v2DataEngineLogFlags and v2DataEngineLogLevel

### PR DESCRIPTION
**Pull request by derekbit**
#### Which issue(s) this PR fixes:

Issue Longhorn/longhorn#7939

### CODE REVIEW

**Comment by derekbit** [comment]
cc @innobead

**Review by shuo-wu — DISMISSED** [review]

File: chart/README.md [hunk 1]
```diff
@@ -322,6 +322,8 @@ During installation, you can either allow Longhorn to use the default system set
 | defaultSettings.v2DataEngine | Setting that allows you to enable the V2 Data Engine, which is based on the Storage Performance Development Kit (SPDK). The V2 Data Engine is a preview feature and should not be used in production environments. |
 | defaultSettings.v2DataEngineGuaranteedInstanceManagerCPU | Number of millicpus on each node to be reserved for each Instance Manager pod when the V2 Data Engine is enabled. The default value is "1250". |
 | defaultSettings.v2DataEngineHugepageLimit | Setting that allows you to configure maximum huge page size (in MiB) for the V2 Data Engine. |
+| defaultSettings.v2DataEngineLogLevel | This allows users to configure the log level if SPDK target daemon (spdk_tgt) of V2 Data Engine. |
```
Comment (jillian-maroket) [inline chart/README.md:325]:
```suggestion
| defaultSettings.v2DataEngineLogLevel | Setting that allows you to configure the log level of the SPDK target daemon (spdk_tgt) of the V2 Data Engine. |
```

File: chart/README.md [hunk 2]
```diff
@@ -322,6 +322,8 @@ During installation, you can either allow Longhorn to use the default system set
...
+| defaultSettings.v2DataEngineLogFlags | This allows users to configure the log flags if SPDK target daemon (spdk_tgt) of V2 Data Engine. |
```
Comment (jillian-maroket) [inline chart/README.md:326]:
```suggestion
| defaultSettings.v2DataEngineLogFlags | Setting that allows you to configure the log flags of the SPDK target daemon (spdk_tgt) of the V2 Data Engine. |
```

File: chart/questions.yaml [hunk 3]
```diff
@@ -695,6 +695,25 @@ questions:
...
+  - "Disabled"
+  - "Error"
+  - "Warn"
+  - "Notice"
+  - "Info"
+  - "Debug"
+  default: "Notice"
+- variable: defaultSettings.v2DataEngineLogFlags
+  label: V2 Data Engine
+  description: "This allows users to configure the log flags if SPDK target daemon (spdk_tgt) of V2 Data Engine."
```
Comment (jillian-maroket) [inline chart/questions.yaml:713]:
```suggestion
  description: "Setting that allows you to configure the log flags of the SPDK target daemon (spdk_tgt) of the V2 Data Engine."
```

File: chart/questions.yaml [hunk 4]
```diff
@@ -695,6 +695,25 @@ questions:
   group: "Longhorn V2 Data Engine (Preview Feature) Settings"
   type: int
   default: "2048"
+- variable: defaultSettings.v2DataEngineLogLevel
+  label: V2 Data Engine
+  description: "This allows users to configure the log level if SPDK target daemon (spdk_tgt) of V2 Data Engine."
```
Comment (jillian-maroket) [inline chart/questions.yaml:700]:
```suggestion
  description: "Setting that allows you to configure the log level of the SPDK target daemon (spdk_tgt) of the V2 Data Engine."
```

File: chart/values.yaml [hunk 5]
```diff
@@ -321,6 +321,10 @@ defaultSettings:
   disableSnapshotPurge: ~
   # -- Maximum snapshot count for a volume. The value should be between 2 to 250
   snapshotMaxCount: ~
+  # -- This allows users to configure the log level if SPDK target daemon (spdk_tgt) of V2 Data Engine.
```
Comment (jillian-maroket) [inline chart/values.yaml:324]:
```suggestion
  # -- Setting that allows you to configure the log level of the SPDK target daemon (spdk_tgt) of the V2 Data Engine.
```

File: chart/values.yaml [hunk 6]
```diff
@@ -321,6 +321,10 @@ defaultSettings:
...
+  v2DataEngineLogLevel: ~
+  # This allows users to configure the log flags if SPDK target daemon (spdk_tgt) of V2 Data Engine.
```
Comment (jillian-maroket) [inline chart/values.yaml:326]:
```suggestion
  # Setting that allows you to configure the log flags of the SPDK target daemon (spdk_tgt) of the V2 Data Engine.
```

**Review by jillian-maroket — COMMENTED** [review]
Let me know if you have concerns about the changes.

**Comment by derekbit** [comment]
Thanks @jillian-maroket.

File: chart/README.md [hunk 7]
```diff
@@ -322,6 +322,8 @@ During installation, you can either allow Longhorn to use the default system set
 | defaultSettings.v2DataEngine | Setting that allows you to enable the V2 Data Engine, which is based on the Storage Performance Development Kit (SPDK). The V2 Data Engine is a preview feature and should not be used in production environments. |
 | defaultSettings.v2DataEngineGuaranteedInstanceManagerCPU | Number of millicpus on each node to be reserved for each Instance Manager pod when the V2 Data Engine is enabled. The default value is "1250". |
 | defaultSettings.v2DataEngineHugepageLimit | Setting that allows you to configure maximum huge page size (in MiB) for the V2 Data Engine. |
+| defaultSettings.v2DataEngineLogLevel | Setting that allows you to configure the log level of the SPDK target daemon (spdk_tgt) of the V2 Data Engine. |
```
Comment (innobead) [inline chart/README.md:325]: Please remember to update the doc.

**Review by innobead — APPROVED** [review]

**Pull request merged**